lib/lynx/error.rb

```diff
@@ -0,0 +1,3 @@
+module Lynx
+  class Error < StandardError; end
+end
```

Comment:
nit: GitHub indicates this as well - there is no newline at the end of the file. I think you can configure your editor to enforce/add  this as well.